pkg/cloudprovider/aws/apis/v1alpha1/provider_validation.go

```diff
@@ -76,30 +74,10 @@ func (a *AWS) validateSecurityGroups() (errs *apis.FieldError) {
 func (a *AWS) validateTags() (errs *apis.FieldError) {
 	// Avoiding a check on number of tags (hard limit of 50) since that limit is shared by user
 	// defined and Karpenter tags, and the latter could change over time.
-	managedTags := ManagedTagsFor(a.Cluster.Name)
```

Comment:
Shouldn't we be validating this still? Easy is you pull it from ctx

Reply:
yep, we should! I meant to make this PR a WIP since I'd removed this when I was working on the plumbing. Will add back in an update